content/docs/command-reference/config.md

```diff
@@ -281,6 +281,9 @@ be overidden explicitly through CLI arguments or through responses in prompts
   [custom HTML template](/doc/command-reference/plots#html-templates) for
   `dvc plots`. Accepts a path relative to the `.dvc/` folder.
 
+- `plots.auto_open` - if `true`, DVC will automatically open the HTML
+  generated by `dvc plots` commands in a browser. `false` by default
```

Comment:
Q: Will this apply to `dvc exp show --html` (or whatever the final command for PCPs is) ? If so maybe add a check box to update this in https://github.com/iterative/dvc.org/pull/3105 at some point).

Reply:
Also this implies a small update in https://github.com/iterative/dvc.org/pull/3098/files#diff-777292f659a766aaad0a04f44b3ed6eb55770470d3fc8dd3490be2b994e98d68R110 (that's on me)...

Reply:
So now that https://github.com/iterative/dvc.org/pull/3098 is merged maybe we should address this here?